Mathlib/Algebra/Category/GroupCat/Colimits.lean

```diff
@@ -220,7 +221,7 @@ def descMorphism (s : Cocone F) : colimit.{w} F ⟶ s.pt where
   toFun := descFun F s
   map_zero' := rfl
   -- Porting note : in `mathlib3`, nothing needs to be done after `induction`
-  map_add' x y := Quot.induction_on₂ x y fun _ _ => by dsimp [(· + ·)]; rw [←quot_add F]; rfl
+  map_add' x y := Quot.induction_on₂ x y fun _ _ => by dsimp; rw [←quot_add F]; rfl
```

Comment:
I don't understand why Lean would elaborate this as `Add.add` instead of `HAdd.hAdd`?

Reply:
It didn't; the old proof had to unfold` HAdd.hAdd` into `Add.add` in order for other dsimp lemmas to apply. That's no longer necessary.